debian/xenial/kubelet/debian/control

```diff
@@ -10,6 +10,6 @@ Vcs-Browser: https://github.com/kubernetes/kubernetes
 
 Package: kubelet
 Architecture: {{ .DebArch }}
-Depends: iptables (>= 1.4.21), kubernetes-cni ({{ .KubeletCNIVersion }}), iproute2, socat, util-linux, mount, ebtables, ethtool, ${misc:Depends}
+Depends: iptables (>= 1.4.21), kubernetes-cni ({{ .KubeletCNIVersion }}), iproute2, socat, util-linux, mount, ebtables, ethtool, conntrack, ${misc:Depends}
```

Comment:
are we fine adding this dependency for older versions of kubernetes too? or do we need to gate this on k8s 1.14+ (or whatever)?

Reply:
no, it's fine for any k8s version. Should have added it earlier lol